### PR TITLE
Feature/decision ctrl

### PR DESCRIPTION
**Pull request by ZackTully**
Ready to be merged
Tests expected to pass

**Feature or improvement description**
New controller type in `Supervisor`: decision control. Decision control makes four decisions: whether to turn stacks on, whether to turn stacks off, which order to switch stacks on and off, and how to distribute power amongst active stacks. It makes each decision according to one of two policies.

**Related issue, if one exists**
none

**Impacted areas of the software**
`Supervisor.py` and `Stack.py` are the only files with serious changes.

**Additional supporting information**
Some of the existing controller options in `Supervisor.py` are equivalent to decision control with the right set of policies. These redundant controllers will be removed eventually.

### CODE REVIEW

File: electrolyzer/inputs/modeling_schema.yaml [hunk 1]
```diff
@@ -87,10 +87,36 @@ properties:
                         description: Number of electrolyzer Stacks to run in simulation
                     control_type:
                         type: string
-                        enum: [PowerSharingRotation, SequentialRotation, EvenSplitEagerDeg, EvenSplitHesitantDeg, SequentialEvenWearDeg, SequentialSingleWearDeg, BaselineDeg]
+                        enum: [PowerSharingRotation, SequentialRotation, EvenSplitEagerDeg, EvenSplitHesitantDeg, SequentialEvenWearDeg, SequentialSingleWearDeg, BaselineDeg, Decision_ctrl]
```
Comment (camirmas) [inline electrolyzer/inputs/modeling_schema.yaml:90]: this is just a nitpick, but consider using CamelCase to match other names, i.e. `DecisionCtrl`

**Comment by camirmas** [comment]
Looks good to me! Would you mind opening an issue for the redundant controllers, so we can track it in case we don't get to it right away?

**Comment by ZackTully** [comment]
Not passing `test_run_lcoh` test. Values are close but not quite equal.

**Review by genevievestarke — COMMENTED** [review]
@ZackTully Hey Zack, I looked over the code, and I was wondering if the update of self.active (in the run_control function) was updated as well?  I don't see it in the changed files?

**Comment by camirmas** [comment]
@ZackTully if you pull `develop` now the tests should pass

**Pull request merged**